### PR TITLE
refactor(`reward`): clean up code

### LINKED ISSUE #467: Improve quality of code generated from templates
Improve quality from generated code for the `reward` module for the end of Milestone 3

Indication: #217

### PR DESCRIPTION
**Pull request by aljo242**
Closes #467

### What does this PR does?

Minor edits to the `reward` module.  
- spelling
- formatting

Most of the templated code seems okay regarding the rules shown in #217.  This PR might not fully address #467, which might need to be re-opened.

### CODE REVIEW

**Review by Pantani — DISMISSED** [review]
LGMT

File: x/reward/client/cli/query_reward_pool.go [hunk 1]
```diff
@@ -14,7 +14,7 @@ import (
 func CmdListRewardPool() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "list-reward-pool",
-		Short: "list all RewardPool",
+		Short: "list all RewardPools",
```
Comment (lubtd) [inline x/reward/client/cli/query_reward_pool.go:17]:
```suggestion
		Short: "list all reward pools",
```
It was `RewardPool` in the sense of the name of the structure, we can remove the camel case as well

File: x/reward/types/expected_keepers.go [hunk 2]
```diff
@@ -13,14 +13,14 @@ type AuthKeeper interface {
...
-	CoordinatorIDFromAddress(ctx sdk.Context, address string) (id uint64, err error)
+	CoordinatorIDFromAddress(ctx sdk.Context, address string) (uint64, error)
 }
 
 type LaunchKeeper interface {
 	GetChain(ctx sdk.Context, launchID uint64) (launchtypes.Chain, bool)
 }
 
-// AccountKeeper defines the expected account keeper used for simulations (noalias)
+// AccountKeeper defines the expected account keeper used for simulations
```
Comment (lubtd) [inline x/reward/types/expected_keepers.go:23]: We can in this PR to remove the unused keepers `AccountKeeper` and `AuthKeeper`

**Review by lubtd — COMMENTED** [review]
👍

FYI you may have seen in the issue but we have some list of things to improve from Starport generation from previous tasks https://github.com/tendermint/spn/issues/217

**Comment by aljo242** [comment]
I removed the `Authkeeper` but am keeping the `Accountkeeper` because it will eventually be used in the simulation tests as shown [here](https://github.com/tendermint/spn/blob/7451726bee345c16ede09f2aca304733bf239a6b/x/launch/simulation/simulation.go#L52)

File: x/reward/client/cli/query_reward_pool.go [hunk 3]
```diff
@@ -14,7 +14,7 @@ import (
 func CmdListRewardPool() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "list-reward-pool",
-		Short: "list all RewardPool",
+		Short: "list all reward pools",
```
Comment (lubtd) [inline x/reward/client/cli/query_reward_pool.go:17]: Can we also set  `"shows a reward pool"` for the query below?

**Review by lubtd — APPROVED** [review]

**Review by giunatale — APPROVED** [review]

**Pull request merged**